.vscode/settings.json

```diff
@@ -1,5 +1,9 @@
 {
   "typescript.tsdk": "node_modules\\typescript\\lib",
   "cSpell.words": ["fluentui", "reactflow"],
-  "eslint.validate": ["json"]
+  "eslint.validate": ["json"],
+  "terminal.integrated.env.windows": {
```

Comment:
nit: remove, we should probably add this file to gitignore, I noticed this was being added to my PRs as well

Reply:
Yeah I agree, I'm just going to add to the .gitignore